src/index.js

```diff
@@ -1,2 +1,70 @@
-angular
-  .module('app', []);
+angular.module('crowdFundApp', ['ng-token-auth', 'ipCookie','ui.router'])
+.config(function($authProvider, $stateProvider, $locationProvider, CONFIG) {
+  $authProvider.configure([{
+    default: {
+      apiUrl: CONFIG.apiUrl
+      // proxyIf: function() { window.isOldIE() },
```

Comment:
All of these commented lines are not useful, we will not authenticate with GitHub anytime soon!